layer/trace_layer.cpp

```diff
@@ -241,6 +271,28 @@ VKAPI_ATTR PFN_vkVoidFunction VKAPI_CALL GetDeviceProcAddr(VkDevice device, cons
     return result;
 }
 
+/**
+ * We don't actually need to do anything for this function,
+ * but we do need to unwrap the instance before the downstream layer
+ * sees it.
+ */
+VKAPI_ATTR PFN_vkVoidFunction VKAPI_CALL GetPhysicalDeviceProcAddr(VkInstance ourInstanceWrapper, const char* pName)
+{
+    PFN_vkVoidFunction result = nullptr;
+
+    if (ourInstanceWrapper != VK_NULL_HANDLE)
+    {
+        const VkInstance              nextLayersInstance = encode::GetWrappedHandle<VkInstance>(ourInstanceWrapper);
+        PFN_GetPhysicalDeviceProcAddr next_gpdpa         = get_next_gpdpa(ourInstanceWrapper);
+        if (next_gpdpa)
```

Comment:
Although I personally prefer to leave off the `!= nullptr` in my own projects because I think it looks better, we've established that we use an explicit comparison with `!= nullptr` in this codebase.  Would you take a moment to add `!= nullptr` here?